### PR TITLE
Useful changes / additions

### PR DESCRIPTION
**Pull request by andyuk1986**

### CODE REVIEW

File: plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java [hunk 1]
```diff
@@ -43,6 +57,20 @@ protected ConfigurationBuilder getDefaultHotRodConfig() {
...
+
+      config.maxRetries(maxRetries).socketTimeout(connectionTimeout).connectionTimeout(connectionTimeout);
+
+      if (propertiesPath != null) {
+         Properties p = new Properties();
+         try (Reader r = new FileReader(propertiesPath)) {
+            p.load(r);
+            config.withProperties(p);
+         } catch (FileNotFoundException e) {
+            e.printStackTrace();
```
Comment (diegolovison) [inline plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java:69]: Should we throw IllegalStateException?

File: plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java [hunk 2]
```diff
@@ -43,6 +57,20 @@ protected ConfigurationBuilder getDefaultHotRodConfig() {
...
+         } catch (IOException e) {
+            e.printStackTrace();
```
Comment (diegolovison) [inline plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java:71]: Should we throw IllegalStateException?

Comment (gustavolira) [reply plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java:71]: nit: if the `catch` are doing the same stuff, probability would be better combine them with `FileNotFoundException | IOException`

File: plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java (hunk 1 above)
Comment (andyuk1986) [reply plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java]: Done!

File: plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java (hunk 2 above)
Comment (andyuk1986) [reply plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java]: @gustavolira @diegolovison it was not possible to combine them, as FileNotFoundException extends IOException (java didn't allow it), so I am just catching IOException and throwing IllegalStateException with message. Hope this ok for both of you :)

Comment (gustavolira) [reply plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java]: Sorry Anna I didn't notice that

**Comment by diegolovison** [comment]
Thanks @andyuk1986

**Pull request merged**